cmd/lakom/app/app.go

```diff
@@ -99,6 +98,9 @@ type Options struct {
 	// UseOnlyImagePullSecrets sets only the image pull secrets of the pod to be used to access the OCI registry.
 	// Otherwise, also the node identity and docker config file are used.
 	UseOnlyImagePullSecrets bool
+	// SkipSignatureVerification configures the webhook to not disallow images without trusted signature.
+	// Instead to deny the request, the webhook will allow it with a warning.
+	SkipSignatureVerification bool
```

Comment:
```suggestion
	// AllowUnverifiedImages configures the webhook to allow images without trusted signature.
	// Such images are not rejected, but instead allowed with a warning.
	AllowUnverifiedImages bool
```

Reply:
Good suggestion, `unverified` is also not the best word, but I will think about it.